.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+on: [pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
+    services:
+      postgres:
+        image: postgres:9.5
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: digitalmarketplace_test
+        # Set health checks to wait until postgres has started
+        options: >-
```

Comment:
I've not seen `>-` before, what does the dash do?

Reply:
I _think_ this is about allowing the expression to continue on to the next line.